src/types/__snapshots__/resolveDescriptors.spec.ts.snap

```diff
@@ -486,6 +486,56 @@ Line 7, col 1:
 "
 `;
 
+exports[`resolveDescriptors should fail descriptors for wf-type-const 1`] = `
+"<unknown>:4:19: Invalid key type for map. Check https://docs.tact-lang.org/book/maps#allowed-types
```

Comment:
why does it say `Invalid key type` and points to a value type?

Reply:
ah, good catch, forgot to remove `key` during refactoring